.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: actionshub/markdownlint@master
 
   ecr:
-    runs-on: [self-hosted]
+    runs-on: kubernetes
```

Comment:
```suggestion
    runs-on: [kubernetes,self-hosted]
```

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow#using-custom-labels-to-route-jobs